src/radical/pilot/agent/agent_0.py

```diff
@@ -41,13 +42,14 @@ class Agent_0(rpu.Worker):
     def __init__(self, cfg, session):
 
         self._uid     = 'agent.0'
-        self._cfg     = cfg
+        self._cfg: ru.Config = cfg
```

Comment:
@andre-merzky Should we start doing hinting? Or should I remove it? 
Or Should I do hinting in method signature?

Reply:
We are not consistent with type hints - but if you use them, then yes, the method signature would be the preferred place to do so.